update docs to reflect new SSC UI (#1512)
semgrep/semgrep-docs: docs/semgrep-supply-chain/triage-remediation.md

<dd>This tab displays information about all of your dependencies across all onboarded repositories.</dd>
</dl>

![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulns.png)
_Figure 1_. Semgrep Supply Chain Vulnerabilities page.

## Assess and triage dependency findings and usages

:::info Prerequisite

> At least one repository that scans for dependencies through Semgrep Supply Chain. See [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).

Reviewer: Link text (title) needs updating to "scan third-party dependencies" (http://semgrep.dev/docs/semgrep-supply-chain/getting-started)

### Filters

Use filters to narrow down your results. The following criteria are available for filtering:

| Filter                 | Description  |
| ---------------------  | ------------ |
| **Projects**           | Filter by repositories connected to your Semgrep account. |
| **Branch**             | Filter by findings in different Git branches. |

> | **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |

Reviewer: ```suggestion
| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with [Teams](/deployment/teams#teams-beta) enabled. |
```
**Teams** is the term that is used in the app itself. AFAIK there's no mention of RBAC :)

| **Tags**               | Filter for findings based on the tags associated with the project. |

> | **Status**             | Filter the triage state of a finding: <ul><li>New: Vulnerabilities that have not undertaken triage or remediation action.</li><li>Fixed: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as Fixed.</li><li>Ignored: Vulnerabilities that have been triaged as ignored by the user. </li></ul>|

Reviewer: ```suggestion
| **Status**             | Filter the triage state of a finding: <ul><li>**New**: Vulnerabilities that have not undertaken triage or remediation action.</li><li>**Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as Fixed.</li><li>**Ignored**: Vulnerabilities that have been triaged as ignored by the user. </li></ul>|
```

| **Severity**           | Filter by the severity of a finding. Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the GitHub Advisory Database.  |
| **Transitivity**       | Filter by the transitivity of the finding. |

> | **Reachability**       | Filter by exposure, or whether the finding is reachable or not. <ul><li>**Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition</li><li>**Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.</li><li>**Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.</li><li>**Unreachable**: A finding is unreachable if you don't use the vulnerable piece of code of the library or package imported</li><li>**Undetermined**: A finding is undetermined if Semgrep Supply Chain determines that you use a vulnerable version of a dependency, but it doesn't have a relevant reachability rule</li></ul>|

Reviewer: ```suggestion
| **Reachability**       | Filter by exposure, or whether the finding is reachable or not. <ul><li>**Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition</li><li>**Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.</li><li>**Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.</li><li>**Unreachable**: A finding is unreachable if you don't use the vulnerable piece of code of the imported library or package.</li><li>**Undetermined**: A finding is undetermined if Semgrep Supply Chain determines that you use a vulnerable version of a dependency, but it doesn't have a relevant reachability rule.</li></ul>|
```
